discord/embeds.pyi

```diff
@@ -0,0 +1,65 @@
+import datetime
+
+from .colour import Colour
+from .types import RawEmbedDict, EmbedFooterData, EmbedImageData, EmbedVideoData, EmbedProviderData, EmbedAuthorData, EmbedFieldData
+
+from typing import Any, Union, Dict, List, ClassVar, TypeVar, Type
+from mypy_extensions import TypedDict
+from typing_extensions import Final
+
+class _EmptyEmbed:
+    def __bool__(self) -> bool: ...
+    def __repr__(self) -> str: ...
+    def __len__(self) -> int: ...
+
+EmptyEmbed: Final[_EmptyEmbed] = ...
+
+class EmbedProxy:
+    def __init__(self, layer: Dict[str, Any]) -> None: ...
+    def __len__(self) -> int: ...
+    def __repr__(self) -> str: ...
+    def __getattr__(self, attr: str) -> _EmptyEmbed: ...
+
+_E = TypeVar('_E', bound=Embed)
+
+class Embed:
+    title: Union[str, _EmptyEmbed]
+    type: str
+    description: Union[str, _EmptyEmbed]
+    url: Union[str, _EmptyEmbed]
+    colour: Union[int, Colour, _EmptyEmbed]
```

Comment:
`Embed.colo(u)r` and `Embed.timestamp` are properties rather than attributes, but this doesn't seem to cause an issue, so I guess this can pass.

Reply:
See comment about `Client.activity`.